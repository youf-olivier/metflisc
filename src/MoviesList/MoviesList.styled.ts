import { createMuiTheme, TextField } from '@material-ui/core';
import { grey } from '@material-ui/core/colors';
import { Link } from 'react-router-dom';
import { device } from 'shared/utilities/devise.constants';

import styled from 'styled-components';

export const StyleMoviesContainer = styled.div`
  display: flex;
  flex-wrap: wrap;
  padding: 0.2em;
  @media ${device.tablet} {
    max-width: 950px;
    margin: auto;
  }
`;

export const StyledLink = styled(Link)`
  display: contents;
`;

export const StyleImg = styled.img`
  width: 48%;
  margin: 0.2em;
  @media ${device.tablet} {
    width: 24%;
  }

  @media ${device.laptop} {
    width: 18%;
  }
`;

export const StyledTextfield = styled(TextField)`
  background-color: #ffffff;
  margin-top: 0.5em;
`;

export const theme = createMuiTheme({
  palette: {
    primary: grey,
  },
});
