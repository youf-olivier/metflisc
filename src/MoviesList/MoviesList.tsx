import { FC } from 'react';

type MoviesListProps = {};

const MoviesList: FC<MoviesListProps> = () => <span>Page DÃ©tail</span>;

export default MoviesList;
