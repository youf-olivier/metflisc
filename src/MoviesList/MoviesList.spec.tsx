import { render } from '@testing-library/react';

import MoviesList from './MoviesList';

describe('MoviesList tests suite', () => {
  it('should render without crashing', () => {
    const { asFragment } = render(<MoviesList />);
    expect(asFragment()).toMatchSnapshot();
  });
});
