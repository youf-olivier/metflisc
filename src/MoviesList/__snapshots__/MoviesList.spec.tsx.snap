// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviesList tests suite should render without crashing 1`] = `
<DocumentFragment>
  <span>
    Page DÃ©tail
  </span>
</DocumentFragment>
`;
